tests/test_load.py

```diff
@@ -0,0 +1,49 @@
+import sqlite3
+from pathlib import Path
+from typing import Generator
+
+import pytest
+
+from etl.load import create_fund_positions_table, load_csvs_into_db, run_sql_script
+
+
+@pytest.fixture
+def db_connection(tmp_path: Path) -> Generator[sqlite3.Connection, None, None]:
+    db_path = tmp_path / "test.db"
+    conn = sqlite3.connect(db_path)
+    yield conn
+    conn.close()
+
+
+def test_run_sql_script(db_connection: sqlite3.Connection, tmp_path: Path) -> None:
+    sql_file = tmp_path / "init.sql"
+    sql_file.write_text("CREATE TABLE test_table (id INTEGER PRIMARY KEY);")
```

Comment:
For small snippets of SQL like this, why write a `.sql` file? Why not just run the SQL query directly?